public/app/features/alerting/unified/components/rule-viewer/AlertRuleMenu.tsx

```diff
@@ -167,6 +170,9 @@ const AlertRuleMenu = ({
       )}
       {/* TODO Migrate Declare Incident to plugin links extensions */}
       {shouldShowDeclareIncidentButton && <DeclareIncidentMenuItem title={promRule.name} url={''} />}
+      {shouldShowAnalyzeRuleButton && prometheusRuleType.alertingRule(promRule) && (
+        <AnalyzeRuleButton alertRule={promRule} />
+      )}
```

Comment:
I think we should limit the button to only GMA rules?
Also, I think that analyzing GMA recording rules should be supported